packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/main/java/io/kaoto/camelcatalog/CamelCatalogProcessor.java

```diff
@@ -36,11 +37,14 @@ public class CamelCatalogProcessor {
     private final ObjectMapper jsonMapper;
     private final DefaultCamelCatalog api;
     private final CamelYamlDslSchemaProcessor schemaProcessor;
+    private final ArrayList<String> patternBlacklist;
```

Comment:
```suggestion
    private final ArrayList<String> patternBlocklist;
```

Reply:
Thank you for bringing this up. I fixed that.